glossary.yml

```diff
@@ -1681,7 +1692,17 @@
       location, such as the [current working
       directory](#current_working_directory). A relative path is the
       equivalent of giving directions using terms like "straight" and
-      "left".  ref: - absolute_path
+      "left".
+    ref: - absolute_path
```

Comment:
Looks like the build is generating an error in this line

Reply:
Thank you so much. I thought it was wrong, I'll  move it back!

Reply:
You are welcome! I'm still trying to understand the patterns of yaml. 

In the lines 1704 and 1705 it looks the same pattern and its working. 

```
  ref: 
    - absolute_path
```

Reply:
actually, first I'll try to emulate other entries that have ref at the end on the same indentation level as languages.